gammapy/obs/observation.py

```diff
@@ -128,6 +141,8 @@ def select_range(self, selection_variable, value_range, inverted=False):
 
         # build and apply mask
         mask = (value_range[0] <= value) & (value < value_range[1])
+        if np.allclose(value_range[0], value_range[1]):
+            mask = value_range[0] == value
```

Comment:
I know, but I think both decisions make sense.
Superfluous parentheses should only be inserted where it improves readability.
Here with `=` and `==` there's possibility to get confused what this statement does.
This is a borderline call, sometimes in complex expressions where operator precedence matters it's more important to insert "superfluous" parentheses, i.e. ones that are only relevant for humans.

Plus: I'm only allowed to make superficial, quick to address suggestions on this PR, so that's what I'm doing. :smile: